docs/scenario-dashboard.md

```diff
@@ -0,0 +1,244 @@
+configurations, and weather configurations. On the bottom right corner of the menu, a user may 
+use "Select Configurations" to make and/or edit selections.
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_starting_menu.png"/>
+  </div>
+
+
+### Vehicles & Sensors
+
+Users can select multiple vehicle and sensor configurations to run. Select a vehicle configuration 
+from monoDrive's vehicle models or add a custom vehicle as the ego vehicle for the simulation. 
+The custom vehicle requires the directory path to the vehicle asset and enables users to change the 
+color of the vehicle. By default all monoDrive's vehicle models spawn with white paint; if a user 
+would like to customize the color on a monoDrive vehicle asset, select "Add New Vehicle" and use 
+the path to the monoDrive vehicle asset and change it's color.
+
+<div class="img_container" >
+  <video width=650px height=480px muted autoplay loop>
+    <source src="https://cdn.monodrive.io/readthedocs/dashboard_vehicle_selection.mp4" type="video/mp4">
+  </video>
+</div> 
+
+For each vehicle selected, a user is given an option to create a new sensor configuration through 
+the sensor editor tool.
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_sensors.png"/>
+  </div>
+
+#### Sensor Editor
+
+The Sensor editor tool enables the user to import a JSON sensor configuration file or build a sensor 
+configuration by adding and moving sensors on the vehicle model. Users are able to alter the 
+position and location for every sensors. Many sensors offer further alterations depending on the 
+sensor type such as image height, image width, and field of view. The "Import Config or Vehicle" 
+Button enables users to import a vehicle model and also can import a sensor json file for easy 
+uploading. After making all selections, the user needs to "Save Config" in the top left corner to 
+save the configuration.
+
+<div class="img_container">
+  <video width=650px height=480px muted autoplay loop>
+    <source src="https://cdn.monodrive.io/readthedocs/dashboard_sensor_editor.mp4" type="video/mp4">
+  </video>
+</div> 
+
+### Unit Under Test (UUT) & Scenarios
+
+ In the monoDrive Batch Scenario Manager, users will be able to run and configure a Unit Under Test. 
+ The Unit Under Test will be configured for closed loop or replay mode. monoDrive has many pre-built 
+ trajectory files to run using the "Straight 5K" Map.
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_scenarios.png"/>
+  </div>
+
+For closed loop mode, the user can add as many scenario JSON files to run in the batch. 
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_closed_loop.png"/>
+  </div>
+
+For details on writing a Unit Under Test and/or work with examples, see 
+[Configuring UUT](/scenario-dashboard/#connecting-your-code).
```

Comment:
this is currently a dead link. not sure if it will work properly when actually deployed?